python/train.py

```diff
@@ -18,61 +18,70 @@
 from __future__ import print_function
 
 import math
+from ffcc import model
+from ffcc import ops
+from ffcc import io
+from ffcc import input as ffcc_input
+import os
+import json
+from tensorboard.plugins.hparams import api as hp
+import tensorflow as tf
 
-from . import model
-from . import ops
-import input as ffcc_input
-import tensorflow.compat.v1 as tf
 
-# This directory stores the TFRecord for training.
-DATA_DIR = ''
+os.environ['CUDA_VISIBLE_DEVICES'] = '-1'
```

Comment:
This new line warrants a comment. What are we doing this for? Probably best not to clobber os.environ, which is global to your entire python process if you can avoid it, although it's fine if there's no other way to achieve this.